app/models/chargeback.rb

```diff
@@ -21,6 +21,7 @@ def self.build_results_for_report_chargeback(options)
     rate_cols = ChargebackRate.where(:default => true).flat_map do |rate|
       rate.chargeback_rate_details.map(&:metric).select { |metric| perf_cols.include?(metric.to_s) }
     end
+    rate_cols.map! { |x| (x == "v_derived_cpu_total_cores_used") ? "cpu_usage_rate_average" : x }
```

Comment:
`v_derived_cpu_total_cores_used`` isn't a column but it is a virtual column of MerticRollup. What's the issue with using it? Because `v_derived_cpu_total_cores_used` and `cpu_usage_rate_average` are definitely different values.